minetest.conf.example

```diff
@@ -1107,7 +1107,7 @@
 
 #    Clickable weblinks (middle-click or Ctrl+left-click) enabled in chat console output.
 #    type: bool
-# clickable_chat_weblinks = false
```

Comment:
Need to update builtin/settingtypes.txt

Reply:
[This commit](https://github.com/Froggo8311/minetest/commit/72fc1acfe52b45386446b66439059228117fb38d) should take care of that.